src/app/hooks/useTranscriptStage/index.tsx

```diff
@@ -0,0 +1,87 @@
+import useOperaMiniDetection from '#app/hooks/useOperaMiniDetection';
+import { Services } from '#app/models/types/global';
+import { useContext, useEffect, useState } from 'react';
+import { ServiceContext } from '../../contexts/ServiceContext';
+
+// Disabled due to bug in ts lint
+// eslint-disable-next-line no-shadow
+export enum Stages {
+  STAGE_1 = `No js stage`,
+  STAGE_2 = `Qualifies for transcript`,
+  STAGE_3 = `Does not qualify for transcript`,
```

Comment:
Would it be possible to include what the stage means in the stage name?
```suggestion
  STAGE_1_NO_JS = `No js stage`,
  STAGE_2_TRANSCRIPT = `Qualifies for transcript`,
  STAGE_3_NO_TRANSCRIPT = `Does not qualify for transcript`,
```

Reply:
If the values were self documenting, could that mean that we don't need an enum anymore?